tests/intergration/test_stream.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2016-2021 Memgraph Ltd. [https://memgraph.com]
```

Comment:
Update to 2022 because that's s new file :D (the same applies to the other PRs)

Reply:
We have to update many files, I think it's easier to do it once all PR's are merged in an "bump year" PR